### DISCUSSION TITLE
4 Textzeilen in deutsch (german)

### DISCUSSION THREAD

**Discussion by nibla-git** [aclap-dev/video-downloadhelper#867] [Language & Translation]
Guten Tag

Ich habe 4 Textzeilen übersetzt.
[de.txt](https://github.com/user-attachments/files/15815573/de.txt)

**Comment by paulrouget** [comment]
Sorry we already got a translation for that language 🙏

I recommend that you try the upcoming version (v9.1 which should be released next week) see if the translation is correct.

If you want to keep track of the current translation status of your language, I recommend following this github repository: https://github.com/aclap-dev/vdhlocales
